index.md

```diff
@@ -20,6 +20,7 @@ title: Testing
 
 1.  [Basics of Testing](01-basics.html)
 2.  [Assertions](02-assertions.html)
+2a. [Design By Contract](02a-dbc.html)
```

Comment:
This does not render properly in markdown.

![image](https://cloud.githubusercontent.com/assets/393899/25724050/04485fa0-30e0-11e7-9f52-98630a12423d.png)

Maybe this section would be appropriate after exceptions or after unit tests anyway?